package.json

```diff
@@ -31,9 +30,12 @@
     "uglify-js": "^2.4.16",
     "watchify": "^2.2.1"
   },
+  "peerDependencies": {
+    "coffeeify": "^1.0.0"
```

Comment:
Try `npm link` to link it globally, then `npm link panoptes` in another directory.

You don't need coffeeify unless you're requiring this as a lib in another project.